structureprovider/nsi-structure-provider.go

```diff
@@ -47,13 +47,15 @@ func InitNSISP() nsiStreamProvider {
 	otp := structures.JsonOccupancyTypeProvider{}
 	otp.InitDefault()
 
+	// TODO probably don't hard code a possibly changing url
```

Comment:
we should use the HEC FWLINK url to obscure the location

Reply:
Probably should get it from a .gitignore configuration file and wipe that info from this public repo

Reply:
fair enough